src/main/scala/io/citrine/lolo/bags/BaggedTrainingError.scala

```diff
@@ -0,0 +1,22 @@
+package io.citrine.lolo.bags
+
+sealed trait BaggedTrainingError {
```

Comment:
I think that this trait should extend `Exception` directly.

You can follow the usual pattern of making this a `sealed abstract class BaggedTrainingError(msg: String)` such that the message can be passed along to the base exception class.

Reply:
I realized something similar while testing locally, although I made it work by extending Throwable. Switching to sealed abstract class now...